### PR TITLE
Integrates webpack-isomorphic-tools to enable asset loading

### LINKED ISSUE #4: Image Requires
Nice work on this example, really interesting!

Experimenting locally and I can't find a solution for universal image requires using this setup.

I've worked with WebpackIsomorphicLoader to solve this problem but the `css-modules-require-hook` seems so elegant, is there an approach similar to this for getting generated image filenames from a require?

### PR DESCRIPTION
**Pull request by adri**
Replaces css-modules-require-hook with [webpack-isomorphic-tools](https://github.com/halt-hammerzeit/webpack-isomorphic-tools).

This avoids having to duplicate the way class names are constructed for css-modules and enables asset loading.

Fixes #4.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -26,6 +16,12 @@ app.use(require("webpack-dev-middleware")(compiler, {
 }));
 app.use(require("webpack-hot-middleware")(compiler));
 
+//  Create hot-reloading bundle for server rendering
+var basePath = require('path').resolve(__dirname, '.');
+var webpackIsomorphicTools = new WebpackIsomorphicTools(require('./webpack-isomorphic-tools'))
+    .development(true)
+    .server(basePath, function() { });
```
Comment (glenjamin) [inline server.js:23]: don't we need to do something in this callback to use the new bundle?

The `webpack-isomorphic-tools` readme mentions `.refresh()`, do we need to call it?

File: webpack.config.js [hunk 2]
```diff
@@ -47,12 +51,16 @@ module.exports = {
...
+      test: /\.css$/,
       include: path.join(__dirname, 'client'),
-      loader: 'style-loader!css-loader?modules&importLoaders=1&localIdentName=[path][name]-[local]'
-    }
+      loader: 'style!css?modules&importLoaders=1&localIdentName=[path][name]-[local]'
+    },
 
+    // Image
+    {
+      test: /\.(png|jpg|jpeg|png|gif|svg)$/, loader: 'url?limit=8192'
```
Comment (glenjamin) [inline webpack.config.js:62]: maybe also include an `` tag in the component, instead of just the CSS?

File: server.js (hunk 1 above)
Comment (glenjamin) [inline server.js:23]: It's also not clear to me what's actually in this bundle, can you expand the comment please?

Comment (adri) [inline server.js:23]: `.refresh()` we don't need because the `require` cache is already cleared further down in this file.

Comment (adri) [inline server.js:23]: Yeah it feels quite magic.

As I understand it takes the assets generated by webpack and maps them correctly when requiring them. 

Do you maybe know a better way to do this? I could imagine even reusing the existing `compiler`.

Comment (glenjamin) [inline server.js:23]: Not tried this myself yet, I'd rather not put something as magic as this in the example if we can't explain it better yet.

Comment (adri) [inline server.js:23]: No problem :)

Comment (glenjamin) [inline server.js:23]: Doesn't mean I wouldn't take it once we figure out what's going on! Maybe the creator of that module will help? Failing that, we'll have to poke around the source a bit

**Pull request closed without merging**